.circleci/config.yml

```diff
@@ -512,6 +512,21 @@ jobs:
             bundle exec fastlane android verify_debug_view_snapshot_tests
       - android/save-build-cache
 
+  emerge_purchases_ui_snapshot_tests:
```

Comment:
We will need to add this job to one of the workflows below. Probably the `build-test-deploy` workflow, so it runs on all commits.

Reply:
Added!